de_web_plugin.cpp

```diff
@@ -19383,6 +19383,14 @@ void DeRestPluginPrivate::genericDisconnectNetwork()
     DBG_Printf(DBG_INFO_L2, "networkState: DisconnectingNetwork\n");
 
     apsCtrl->setNetworkState(deCONZ::NotInNetwork);
+    
+    if (!reconnectTimer)
+    {
+        reconnectTimer = new QTimer(this);
+        reconnectTimer->setSingleShot(true);
+        connect(reconnectTimer, SIGNAL(timeout()),
+                this, SLOT(reconnectTimerFired()));
+    }
```

Comment:
Good catch but I think the following is more appropriate since there are more state variables involved, the method will also initialize the timer if not already created:

```cpp
// reconnectTimer->start(DISCONNECT_CHECK_DELAY);

startReconnectNetwork(RECONNECT_CHECK_DELAY);
```

Reply:
Was tempted to use that, but wasn't too sure since I don't know the parts of the code too well. Hence, a more conservative initial approach ;)